bagel/utilities/pheno_utils.py

```diff
@@ -24,10 +24,13 @@
 }
 
 
-def validate_portal_uri(portal: str) -> Optional[str]:
+def validate_portal_uri(portal: Optional[str]) -> Optional[str]:
```

Comment:
was that just wrong before? Are we allowing `None` now?

Reply:
yeah, the type was just wrong before - `none` was always an allowed value, because `--portal` itself is optional!